### ISSUE TITLE
new release

### ISSUE THREAD

**Issue by paztis** [Leonidas-from-XIV/node-xml2js#517]
Hi all

Dependencies versions have been updated 1 year ago, but last xml2js release is still 2 years old
Is it possible to release it again ?

In last npm release, xmlbuilder is in version ~9.0.1
In last git package.json, xmlbuilder is in version ~10.0.0
and more xmlbuilder is now in version 13.0.2

xmlbuilder adds now the ability to format attributes on multiple lines that is missing in xml2js
Is it possible to increase the version and republish it ?

Thanks

**Comment by paztis** [comment]
or better, to let the ability in the Builder contructor to pass another instance of xmlbuilder.
Like this people like me can pass a more recent version of xmlbuilder without forcing you to update your code. I can create a PR for this if needed

**Comment by knoxcard** [comment]
Exactly...https://github.com/Leonidas-from-XIV/node-xml2js/pull/524

**Comment by knoxcard** [comment]
That would be pretty sweet actually!

```
const xml2js = require('xml2js');

const xml = '<root>Hello world!</root>'

xml2js.parseString(xml, function (err, result) {
    console.log(result.root); // Output: Hello world!
});
```

**Comment by Leonidas-from-XIV** [comment]
I'll try to make a release this weekend.

**Comment by Leonidas-from-XIV** [comment]
Just published 0.4.20.

**Comment by tomhughes** [comment]
Could you tag it on github please?

**Comment by Leonidas-from-XIV** [comment]
Ah yes, forgot to push the commit. There you go, it is tagged now.

**Comment by paztis** [comment]
Version of xmlbuilder is not increased to ~13.0.0 in your release ?

**Comment by Leonidas-from-XIV** [comment]
No, I did not see a reason to do so.

**Comment by paztis** [comment]
I don't know if multiline attributes formatting is available in 10.0.0.
Will check it tomorrow

**Comment by paztis** [comment]
No it is available since 12.0.1 : https://github.com/oozcitak/xmlbuilder-js/commit/c92ba3e6387ab305e9487b4bee546f249da33f4e

It means my defect is still opened, as there's still no way to pass an upper version.

**Comment by Leonidas-from-XIV** [comment]
You have a link to your issue?

**Comment by paztis** [comment]
It is explain in the first post of this topic. That was the reason I ask to increase the dependency version on xmlbuilder.

Currently I'm is not possible to format a xml with attributes on multiple lines. As I work on big xml with long attributes names, result is not readable.

Xmlbuilder allows since the version 12 to pass a "width" option to limit the width or the lines.
As you just pass down the option to xmlbuilder, it works automatically on your module if we're able to move to version 12.0.1 at minimum.

**Comment by Leonidas-from-XIV** [comment]
https://github.com/Leonidas-from-XIV/node-xml2js/pull/528

**Comment by Leonidas-from-XIV** [comment]
Released as 0.4.21 just now.

**Comment by paztis** [comment]
Perfect thanks a lot
